packages/ui/src/stories/molecules/VoteCounter.stories.tsx

```diff
@@ -21,6 +27,10 @@ const MockedVoteCounter: FC<TVoteCounter> = ({ voteCount }) => {
         userVote: voteCountUpdated.userVote + 1,
       }
       setVoteCountUpdated(newVoteUp)
+      setIsUpVoteDisabled(true)
+      setTimeout(() => {
+        setIsUpVoteDisabled(false)
+      }, 1000)
```

Comment:
I have a doubt. Why do you use the setTimeout()? If the user clicks and the button is set disabled, it shouldn't need to go back to not being disabled no?